microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -91,3 +91,39 @@ groups:
         title: "Metadata Overhead Above 90% on Bucket: {{ $labels.bucket }}, Node: {{ $labels.instance }}"
         description: The percentage of memory that is taken up by metadata is over 90%
         remediation: Increase memory allocation for bucket or change the evictionPolicy of the bucket from `Value-only` (be aware this will have an adverse effect on performance).
+
+    - alert: CB90061-diskWriteQueueLength-Alert
+      expr: |
+        kv_ep_storage_age_seconds > 100
```

Comment:
Needs to be >99 or change the warn alert, I know it'll just take a second longer to fire but good to be precise

Reply:
I've changed it to 99